src/partition.rs

```diff
@@ -412,8 +414,61 @@ pub enum MvJointType {
   MV_JOINT_HNZVNZ = 3  /* Both components nonzero */
 }
 
+fn scale_chroma_bsize(bsize: BlockSize, ss_x: usize, ss_y: usize)
```

Comment:
So this is the inverse of BlockSize::subsampled_size? Might be nice to put it next to that and call it something like BlockSize::supersampled_size?

Reply:
Done.